modules/backend/lang/pl/lang.php

```diff
@@ -1,568 +1,627 @@
+        'return_to_parent' => 'Wróć do folderu nadrzędnego',
+        'return_to_parent_label' => 'Idź w górę ..',
+        'nothing_selected' => 'Nic nie zaznaczono.',
+        'multiple_selected' => 'Wybrano wiele elementów.',
+        'uploading_file_num' => 'Przesyłanie :number pliku(ów)...',
+        'uploading_complete' => 'Przesyłanie zakończone',
+        'uploading_error' => 'Błąd przesyłania pliku',
+        'type_blocked' => 'Typ pliku jest zablokowany ze względów bezpieczeństwa.',
+        'order_by' => 'Sortowanie',
+        'direction' => 'Kierunek',
+        'direction_asc' => 'Rosnąco',
+        'direction_desc' => 'Malejąco',
+        'folder' => 'Folder',
+        'no_files_found' => 'Nie znaleziono żadnych pasujących elementów.',
+        'delete_empty' => 'Wybierz elementy do usunięcia.',
+        'delete_confirm' => 'Czy na pewno chcesz usunąć wybrane elementy?',
+        'error_renaming_file' => 'Wystąpił błąd przy zmianie nazwy elementu.',
+        'new_folder_title' => 'Nowy folder',
+        'folder_name' => 'Nazwa folderu',
+        'error_creating_folder' => 'Wystąpił błąd przy tworzeniu folderu',
+        'folder_or_file_exist' => 'Folder lub plik o podanej nazwie już istnieje.',
+        'move_empty' => 'Wybierz elementy do przeniesienia.',
+        'move_popup_title' => 'Przenieś pliki lub foldery',
+        'move_destination' => 'Folder docelowy',
+        'please_select_move_dest' => 'Wybierz folder docelowy.',
+        'move_dest_src_match' => 'Wybierz inny folder docelowy.',
+        'empty_library' => 'Biblioteka Mediów jest pusta. Na początek prześlij pliki lub utwórz foldery.',
+        'insert' => 'Wstaw',
+        'crop_and_insert' => 'Przytnij lub wstaw',
+        'select_single_image' => 'Wybierz pojedynczy obrazek.',
+        'selection_not_image' => 'Wybrany element nie jest obrazkiem.',
+        'restore' => 'Cofnij wszystkie zmiany',
+        'resize' => 'Skaluj...',
+        'selection_mode_normal' => 'Normalny',
         'selection_mode_fixed_ratio' => 'Zablokuj proporcje',
-        'selection_mode_fixed_size'  => 'Zablokuj rozmiar',
-        'height'                     => 'Wysokość',
-        'width'                      => 'Szerokość',
-        'selection_mode'             => 'Tryb zaznaczenia',
-        'resize_image'               => 'Zmień rozmiar obrazka',
-        'image_size'                 => 'Rozmiar obrazka:',
-        'selected_size'              => 'Wybrano:',
-        'direction'                  => 'Kierunek',
-        'direction_asc'              => 'Rosnąco',
-        'direction_desc'             => 'Malejąco',
+        'selection_mode_fixed_size' => 'Zablokuj rozmiar',
+        'height' => 'Wysokość',
+        'width' => 'Szerokość',
+        'selection_mode' => 'Tryb zaznaczenia',
+        'resize_image' => 'Zmień rozmiar obrazka',
+        'image_size' => 'Rozmiar obrazka:',
+        'selected_size' => 'Wybrano:',
     ],
-];
+];
```

Comment:
@tomaszstrojny Our code checks required the last line of each PHP file to be an empty line.
```suggestion
];

```

Reply:
Nah, sorry about that, I have pushed the translations from a different PR to this branch... https://github.com/octobercms/october/pull/5129